website/project/model.py

```diff
@@ -1604,7 +1604,7 @@ def update(self, fields, auth=None, save=True):
                     {
                         'id': value.get('id'),
                         'year': value.get('year'),
-                        'copyrightHolders': value.get('copyrightHolders')
+                        'copyrightHolders': value.get('copyrightHolders') or value.get('copyright_holders', [])
```

Comment:
It's essentially just a bit of a mess between JS, python, and a model property that's in camel case...